.github/workflows/patch-release-build.yml

```diff
@@ -137,6 +138,7 @@ jobs:
           platforms: linux/amd64,linux/arm64
           tags: |
             public.ecr.aws/aws-observability/adot-autoinstrumentation-java:v${{ github.event.inputs.version }}
+          driver: docker-container
```

Comment:
driver is not a input option for `build-push-action` but only for `setup-buildx-action`
https://github.com/docker/build-push-action?tab=readme-ov-file#inputs

https://github.com/docker/setup-buildx-action?tab=readme-ov-file#customizing